### PR TITLE
Add smoke test and notification

### PR DESCRIPTION
**Pull request by jsugarman**
#### What
Add smoke test of application with team notification

#### Ticket

[CBO-815](https://dsdmoj.atlassian.net/browse/CBO-815)

#### Why
To sanity/smoke test the `actual` built app
to ensure it passes fundamental functional checks,
over and above the test suite.

#### How
use circleCI to run the smoke test
against a container based on the latest
master branch's image from ECR.

Use slack orb plus some customization to
notify the team

--------

#### TODO (wip)

 - [X] smoke test running against latest image
 - [X] smoke test notifies success and failure
 - [ ] cronjob to run smoke test at 9am each morning
 - [ ] additional master branch merge step to run smoke test

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -18,11 +18,7 @@ devunicorn:
 
 test: &test
   <<: *default
-  database: <%= ENV['CBO_BASE_DATABASE_DATABASE'] || 'cbo_test' %>
```
Comment (jsugarman) [inline config/database.yml:21]: just removing superfluos config. the `develop` branch no longer exists but did in intial stages of the project

File: .circleci/config.yml [hunk 2]
```diff
@@ -329,3 +429,18 @@ workflows:
       - deploy-production:
           requires:
             - hold-production
+
+  scheduled-smoke-test:
+    # triggers:
```
Comment (jsugarman) [inline .circleci/config.yml]: will enable this once i can tell smoke test is succeeding and failing properly on master

File: .circleci/config.yml [hunk 3]
```diff
@@ -110,24 +113,51 @@ executors:
...
         REPO_NAME: cccd
 
+  smoke-test-executor:
+    working_directory: /usr/src/app
+    docker:
+      - image: ${ECR_ENDPOINT}/laa-get-paid/cccd:app-latest
+        entrypoint: |
+          docker/docker-entrypoint.sh
+        environment:
+          BASH: true
```
Comment (jsugarman) [inline .circleci/config.yml:123]: note to self, i think i can remove this now.

**Review by kmahern — APPROVED** [review]
👍

**Review by kmahern — APPROVED** [review]
👍

**Pull request merged**